packages/block-editor/src/components/block-list/block.js

```diff
@@ -511,22 +528,13 @@ function BlockListBlock( {
 				rootClientId={ rootClientId }
 			/>
 			{ isFirstMultiSelected && (
-				<BlockMultiControls rootClientId={ rootClientId } />
+				<BlockMultiControls
+					rootClientId={ rootClientId }
+					horizontalMover={ horizontalMover }
+				/>
 			) }
 			<div className="editor-block-list__block-edit block-editor-block-list__block-edit">
-				{ shouldRenderMovers && (
-					<BlockMover
-						clientIds={ clientId }
-						blockElementId={ blockElementId }
-						isHidden={ ! isSelected }
-						isDraggable={
-							isDraggable !== false &&
-							( ! isPartOfMultiSelection && isMovable )
-						}
-						onDragStart={ onDragStart }
-						onDragEnd={ onDragEnd }
-					/>
-				) }
+				{ shouldRenderMovers && blockMover }
```

Comment:
Why we only render vertical movers here? What is the main reason for passing the horizontal movers to the BlockEdit instead of rendering them here?